e2e/components/Token.test.ts

```diff
@@ -2,34 +2,47 @@ import {test, expect} from '@playwright/test'
 import {visit} from '../test-helpers/storybook'
 import {themes} from '../test-helpers/themes'
 
-test.describe('Token', () => {
-  test.describe('Default', () => {
-    for (const theme of themes) {
-      test.describe(theme, () => {
-        test('default @vrt', async ({page}) => {
-          await visit(page, {
-            id: 'components-token--default',
-            globals: {
-              colorScheme: theme,
-            },
-          })
-
-          // Default state
-          expect(await page.screenshot()).toMatchSnapshot(`Token.Default.${theme}.png`)
-        })
+const stories = [
+  {
```

Comment:
If you have the time, would you mind moving the existing stories in the file to this array? Would help out a ton since we're moving to this new format compared to the older one 😓 

Totally understand if not, either works for me!

Reply:
I'll get those changes in!